blocks/collapse/collapse.css

```diff
@@ -1,12 +1,82 @@
-.text-block__expandable-area {
+.section {
+  display: block;
+  flex-direction: column;
```

Comment:
Since we have `display: block;` , I think its irrelevant to have `flex-direction: column;`